lib/ChGovUk/Models/Address.pm

```diff
@@ -64,6 +65,7 @@ sub as_api_hash {
 
     my $api_hash = {};
 
+    $api_hash->{reference_etag} = $self->etag     if $self->etag;
```

Comment:
Is it by design that the from_api_hash() method and as_api_hash() does not 'round-trip' the value for reference_etag?

Reply:
This is because we need to read the current `etag` from the ROA API but will send it back as `reference_etag`